code/game/mecha/mecha_construction_paths.dm

```diff
@@ -198,7 +198,7 @@
 				user.visible_message("[user] removes the wiring from [parent].", "<span class='notice'>You remove the wiring from [parent].</span>")
 		if(5)
 			if(diff==FORWARD)
-				user.visible_message("[user] installs the central control module into [parent].", "<span class='notice'>You install the central computer mainboard into [parent].</span>")
+				user.visible_message("[user] installs the [I.name] into [parent].", "<span class='notice'>You install the [I.name] into [parent].</span>")

```

Comment:
Just needs to be `[I]` and I *think* the `the` before is unnecessary?

Reply:
Oh yeah, `[I]` works. The extra "the" is neccessary, else we'd get messages like `You install Ripley Central Control module (Exosuit Board) into the Ripley chassis.`